herbridge/frontend/src/lib/api.js

```diff
@@ -1,13 +1,22 @@
 import axios from 'axios'
 
 const login = (password) => {
-    return new Promise((resolve, reject) => {
-        axios.post('/api/login', { password })
-            .then(response => resolve(response.data.token))
-            .catch(error => reject(error))
-    })
+  return new Promise((resolve, reject) => {
+    axios.post('/api/login', {password})
+      .then(response => resolve(response.data.token))
+      .catch(error => reject(error))
+  })
+}
+
+const getResources = (polygon) => {
+  return new Promise((resolve, reject) => {
+    axios.post('/api/eamena/resources', polygon)
+      .then(response => resolve(response.data))
+      .catch(error => reject(error))
+  })
```

Comment:
only comment on this PR is the same one from the last, change this to a mapping `then`, basically